client/src/utils.js

```diff
@@ -240,10 +240,7 @@ Promise.prototype.log = function () {
   })
 }
 
-export const renderBlueprintIconAsSvg = (
-  iconName,
-  iconSize: Icon.SIZE_STANDARD
```

Comment:
no typescript in js

Reply:
It should probably be `iconSize = Icon.SIZE_STANDARD` (default parameter assignment if no explicit icon size is passed by the caller).